src/sentry/integrations/jira_server/integration.py

```diff
@@ -59,6 +59,15 @@
     ),
 ]
 
+setup_alert = {
```

Comment:
This needs to only be shown for SaaS since it doesn't apply for on-premise or single-tenant. Or at least it's completely different data which we don't have any way to configure.

Reply:
Yeah. We already have something like this for GHE, so I was just adding it here as well.

We'll need to do something for both

Reply:
Do we have a settings flag that lets us know we're on saas?